pkg/webhook/preflight/nutanix/storagecontainer.go

```diff
@@ -160,10 +160,10 @@ func newStorageContainerChecks(cd *checkDependencies) []preflight.Check {
 		cd.nutanixClusterConfigSpec.ControlPlane.Nutanix != nil {
 		checks = append(checks,
 			&storageContainerCheck{
-				nodeSpec: cd.nutanixClusterConfigSpec.ControlPlane.Nutanix,
-				field:    "cluster.spec.topology[.name=clusterConfig].value.controlPlane.nutanix",
```

Comment:
```suggestion
				field:    "cluster.spec.topology[.name=clusterConfig].value.controlPlane.nutanix.machineDetails",
```
Also requires a corresponding change to unit tests that check the field value.